### ISSUE TITLE
<<-- arrow is not drawn correctly

### ISSUE THREAD

**Issue by devwitalik** [plantuml/plantuml#1678]
```plantuml
@startuml
Test <<-- Test: the arrow is not drawn correctly
Test -->> Test: the arrow is drawn correctly
@enduml
```
http://www.plantuml.com/plantuml/uml/SoWkIImgAStDuGf9BIvHiB7HrLK0iQmKIZ9I5HABYlBB5JAB5VBoInHIYXBBynIIyukAKfDBSYeX6dHrxUomQC1Gx92QbmAq1G00

**Expected behavior**
```plantuml
<<--  the arrow should be thin
```
**Screenshots**
![IMG](https://www.plantuml.com/plantuml/png/SoWkIImgAStDuGf9BIvHiB7HrLK0iQmKIZ9I5HABYlBB5JAB5VBoInHIYXBBynIIyukAKfDBSYeX6dHrxUomQC1Gx92QbmAq1G00)

**Comment by The-Lum** [comment]
Thanks for this report.

See similar request here:
- https://forum.plantuml.net/14942/cannot-change-the-arrow-type-self-message-sequence-diagram

Regards,
Th.

**Comment by jimnelson372** [comment]
I have coded the fix for this, ensuring all the arrows work the same on for the left side self messages as for the right.   Here's the table of all the left side arrow messages being tested, other than those with an x.

![testarrows2_002](https://github.com/plantuml/plantuml/assets/66284321/412bb63e-ee3e-4c04-bb8c-cc63572f8a48)


Here is the test script.  I put Bob to the left, even though it's not used, just to to be a reflection of the right side message display in the documenation:

```
@startuml
!pragma teoz true
participant Bob  as b
participant Alice as a
activate a
activate b
a <-     a : ""<-""
a o<-     a : ""o<-""
a o<-o     a : ""o<-o""
a <<-    a : ""<<-""
a /-     a : ""/-""
a //-    a : ""//""
a \-     a : ""\-""
a \\-    a : ""\\\\-""
a <-o    a : ""<-o""
a o<-    a : ""o<-""
a o<-o   a : ""o<-o""
a o<<-   a : ""o<<-""
a o/-    a : ""o/-""
a o//-   a : ""o//-""
a o\-    a : ""o\\-""
a o\\-   a : ""o\\\\-""
@enduml
@startuml
```
The image above is with "teoz", but trust me it look almost identical with "puma".   I am ready to create a pull request for this change, as well as the changes for #1683.

Though, I do have a question about the X related left arrows such as <-x and o<-x.   It seems they are currently designed to stay on the right side, but lookup back upwards.  In the code, this seems intentional, so I didn't want to immediately make changes to it.  But can someone confirm if this is correct.   Here's the image:

![testarrows2_003](https://github.com/plantuml/plantuml/assets/66284321/d234e0fe-9bb5-4f1e-9353-4a0f98673e30)

The last 3 are examples with <- arrows.  I was just surprised by the final two.  It works this way identically in both "teoz" and "puma", and if you run the following script in the current version of PlantUML:

```
@startuml
!'pragma teoz true
'skinparam sequence {
'ArrowColor Green
'}
participant Bob   as b
participant Alice as a
activate a
activate b

a <->    a : "" <->""
a o<->o  a : "" o<->o""
a x<->x  a : ""  x<->x""
a x->o   a : ""x->o ""
a ->x    a : ""->x  ""
a x->    a : ""x->  ""
a x<-    a : ""x<-""
a <-X    a : ""<-x""
    note right
      Is this upward loop when left arrow starting from X intentional?
      Or should it go down on the left side like the rest?
    endnote
a o<-x   a : "" o<-x ""
    note right
        Same question as for the previous
    endnote
@enduml
@startuml
```

I hope to submit the pull request tomorrow, and if possible I will make any needed changes to this  X arrow issue.

(FYI: @The-Lum and @arnaudroques, please see this question.  Thanks.)

**Comment by jimnelson372** [comment]
I forgot to include an image in my previous comment showing that the arrows are fixed in the example at the top of this issue.  Here's how it will now look:

![testarrows2_006](https://github.com/plantuml/plantuml/assets/66284321/1569f60c-d445-41e8-9ff4-8eb5c838f9e8)

**Comment by arnaudroques** [comment]
> Though, I do have a question about the X related left arrows such as <-x and o<-x. It seems they are currently designed to stay on the right side, but lookup back upwards. In the code, this seems intentional, so I didn't want to immediately make changes to it. But can someone confirm if this is correct

Actually, I don't think this is really intentional. The "x" feature have been implemented quickly and was not completely consistent.
So it is probably time to fix this and to make it consistent with other syntax.

So to me, you can change the actual behaviour. @The-Lum What is your opinion about it?

**Comment by The-Lum** [comment]
@jimnelson372 and @arnaudroques:

> So it is probably time to fix this and to make it consistent with other syntax.

Yes, you can change to have a more consistent behavior.

_Thanks for your work._
Regards,
Th.

**Comment by jimnelson372** [comment]
Okay, I fixed that left side x arrows.  So my example (with just the note changes) now looks like this:

![testarrows3](https://github.com/plantuml/plantuml/assets/66284321/0da356e5-4140-4ef1-934b-6a4d58958fdd)

**Comment by jimnelson372** [comment]
Hi @The-Lum, I believe this issue can be closed as well.  It works correctly in 1.2024.5. 

By the way, is there some way that the first image above, for this issue, now looks correct?   It now correctly shows the left side self message arrow as thin, when it was thick when originally reported (and is thick if generated with versions prior to 1.2024.5).  I'm unclear about the different choices for inserting diagrams into these comments.  Since I've been including diagrams from my test environment, I've generally been dragging SVG files into my comments.

Jim N.

**Comment by The-Lum** [comment]
Hi all,

_[This is an Issue Review]_ 👀 
This is now fixed on [V1.2024.5](https://github.com/plantuml/plantuml/releases/tag/v1.2024.5).
- Thanks @devwitalik for the report.
- Thanks PlantUML team and @jimnelson372 for your work.

Regards.

**Comment by The-Lum** [comment]
Hello @jimnelson372,

For:
> By the way, is there some way that the first image above, for this issue, now looks correct? It now correctly shows the left side self message arrow as thin, when it was thick when originally reported (and is thick if generated with versions prior to 1.2024.5). I'm unclear about the different choices for inserting diagrams into these comments. Since I've been including diagrams from my test environment, I've generally been dragging SVG files into my comments.

It is due to the fact the link is a link to the `plantuml.com` server.
As:
- http://www.plantuml.com/plantuml/svg/SoWkIImgAStDuGf9BIvHiB7HrLK0iQmKIZ9I5HABYlBB5JAB5VBoInHIYXBBynIIyukAKfDBSYeX6dHrxUomQC1Gx92QbmAq1G00

Then during the time, when the server is update... the images are update also...

Then: your including by dragging SVG is a good practice, in order to keep the issue or keep change history...

Regards,
Th.
